modules/order/commerce_order.views.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Provide views data that isn't tied to any other module.
```

Comment:
That's a weird comment. Why not "Provides views data for the Order module." or "Provides views data for orders."?